src/App/Fossa/FossaAPIV1.hs

```diff
@@ -154,9 +154,11 @@ uploadAnalysis rootDir baseUri key ProjectRevision{..} metadata projects = fossa
           <> "v" =: cliVersion
           <> "managedBuild" =: True
           <> "title" =: fromMaybe projectName (projectTitle metadata)
-          <> "branch" =: projectBranch
           <> apiHeader key
           <> mkMetadataOpts metadata
+          <> case projectBranch of
```

Comment:
it's annoying that the `Maybe` instance doesn't behave this way.

[nit] let's use `maybe :: b -> (a -> b) -> Maybe a -> b` here:
```suggestion
          <> maybe mempty ("branch" =:) projectBranch
```

Reply:
I didn't like this code either, thanks for the suggestion.  I forgot to bring it up as a comment.